frontend/app/routes/multiplayer.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  setupController:function(controller,model){
+    this._super(controller, model)
+    controller.set('gameState',[{},{},{},{},{},{},{},{},{}]);
+    controller.set('showStartNewGame', true);
+    controller.set('message', "");
+  },
+  resetController: function(controller,isExisting, transition){
+    this._super.apply(this, arguments);
+    controller.set('message', "")
```

Comment:
If you put the state on the model instead of on the controller, you don't have to reset controller properties. Try to use the `model` function that routes offer